plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostingFun/Core.hs

```diff
@@ -177,51 +192,51 @@ runOneArgumentModel (ModelOneArgumentLinearCost (ModelLinearSize intercept slope
 
 -- | s * (x + y) + I
 data ModelAddedSizes = ModelAddedSizes
-    { modelAddedSizesIntercept :: CostingInteger
-    , modelAddedSizesSlope     :: CostingInteger
+    { modelAddedSizesIntercept :: Intercept
```

Comment:
Maybe we should make all of these model types have a sub-record that describes a linear function via named fields or something, but let's think about that some other time.  This solve the problem for the time being.